launcher/updatedialog_moc.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * updatedialog_moc.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+#include <QDialog>
+
+class JsonNode;
+
+namespace Ui {
+class UpdateDialog;
+}
+
+class UpdateDialog : public QDialog
+{
+	Q_OBJECT
+
+public:
+	explicit UpdateDialog(QWidget *parent = nullptr, bool calledManually = false);
```

Comment:
`parent` param should be last by Qt convention

Reply:
what is QT convention?